docs/demos.rst

```diff
@@ -0,0 +1,99 @@
+Demos
+================
+
+Multi-object using ROS and DOPE with RGB and Depth
+--------------------------------------------------
+
+The demo uses RGB and Depth from a RealSense camera, poses published by DOPE,
+and segments the image using
+`segment-anything <https://github.com/facebookresearch/segment-anything>`_.
+The demo comes as a catkin package under ``examples/``.
+
+Say you cloned the diff-dope repo under your home directory (i.e., ``~/diff-dope``),
+and you have a catkin workspace under your home directory too (i.e., ``~/catkin_ws``),
+you can create a symlink of the package there:
+
+.. code::
+
+    cd ~/catkin_ws/src
+    ln -s ~/diff-dope/examples/diffdope_ros .
+
+You can ``catkin_make`` under ``~/catkin_ws`` now to build the package.
+You can, of course, move the package there instead of creating a symlink.
+
+The demo uses a configuration under
+``diffdope_ros/config/multiobject_with_dope.yaml`` which uses DOPE for initial
+pose estimation, and dictates topics for RGB and Depth (assuming a RealSense
+sensor). It also uses a camera info topic to retrieve camera intrinsic
+parameters as well as the image dimensions.
+
+You can also download a ROS bag from `here <https://leeds365-my.sharepoint.com/:u:/g/personal/scsrp_leeds_ac_uk/Ec-TbyOr1QVIt6NQQP7E4pABkEUmaEGByVjLHugY7Als_A?e=JES96n>`_
```

Comment:
Please consider replacing this link with yours, as this link will expire on 5th of May.